sources/packages/backend/apps/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -61,6 +61,7 @@ export class InstitutionLocationDetailsAPIOutDTO extends AddressDetailsAPIOutDTO
   primaryContactLastName: string;
   primaryContactEmail: string;
   primaryContactPhone: string;
+  isBCPublicInstitution: boolean;
```

Comment:
https://github.com/bcgov/SIMS/pull/3734/files#r1774067185

Reply:
`isBCPublicInstitution` is removed.